source/testday/ussuri/milestone1.html.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: RDO Stein Test Day
+title: RDO Ussuri Test Day
 ---
 
-# Stein Test Day 1
+# Stein Milestone1 Test Day
 
-We will be holding a RDO test day on **01 and 02 November 2018.**
+We will be holding a RDO test day on **19 and 20 December 2019.**
 
 This will be coordinated through the **#rdo channel on Freenode**, and through
 this website and the [dev@lists.rdoproject.org](https://lists.rdoproject.org/mailman/listinfo/dev) mailing list.
 
-We'll be testing the first [Stein milestone release](http://releases.openstack.org/stein/schedule.html). If you can do any testing on your own ahead of time, that will help ensure that everyone isn't encountering the same problems.
+We'll be testing the first [Ussuri milestone release](http://releases.openstack.org/ussuri/schedule.html). If you can do any testing on your own ahead of time, that will help ensure that everyone isn't encountering the same problems.
 
 We will also be available for people that want to try the stable (Rocky) release.
 
 ### Quick links
 
 * [Test matrix](/testday/tests)
-* [Workarounds](https://etherpad.openstack.org/p/rdo-test-days-stein1-workarounds)
+* [Workarounds](https://etherpad.openstack.org/p/rdo-test-days-ussuri1-workarounds)
 
 ## Prerequisites
 
 We plan to have packages for the following platforms:
 
-* [RHEL 7](https://access.redhat.com/products/red-hat-enterprise-linux/)
-* [CentOS 7](https://www.centos.org/download/)
+* [RHEL 8](https://access.redhat.com/products/red-hat-enterprise-linux/)
```

Comment:
I like the idea of having a testday short after having it ready on CentOS8 but as today, we can't commit with having RDO on CentOS8 by Ussuri Milestone 1. We have external dependencies and we'll need some time to have it ready.

Reply:
Fair. Keep it on RHEL / CentOS 8 for Ussuri M3?

Reply:
Hopefully at M3 we should have something for CentOS8.